hack/check-license-header.sh

```diff
@@ -18,7 +18,16 @@ set -e
 
 echo "> Checking if license header is present in all required files."
 
-missing_license_header_files="$(addlicense -check -ignore "vendor/**" -ignore "**/*.md" -ignore "**/*.html" -ignore "**/*.yaml" -ignore "**/Dockerfile" --ignore "hack/tools/gomegacheck/**" --ignore "pkg/operation/botanist/**/*.sh" .)" || true
+missing_license_header_files="$(addlicense -check \
```

Comment:
Can you adapt the same here https://github.com/gardener/gardener/blob/1e9b2c1a4e5ec47d29b250eaf99a35dad05f525d/hack/add-license-header.sh#L23